docs/shared/cypress-plugin.md

```diff
@@ -83,6 +83,101 @@ nx e2e frontend-e2e --baseUrl=https://frontend.com
 
 If you need to fine tune your Cypress setup, you can do so by modifying `cypress.json` in the e2e project. For instance, you can easily add your `projectId` to save all the screenshots and videos into your Cypress dashboard. The complete configuration is documented on [the official website](https://docs.cypress.io/guides/references/configuration.html#Options).
 
+## Code coverage
+1. instrument cypress of your `frontend-e2e` and your frontend framework to use code coverage as described on the official cypress documentation. See below how to insctruct angular. You may also need to check if the setting `pluginsFile` of `frontend-e2e/cypress.json` needs to be updated.
```

Comment:
@barbados-clemens 
Thanks for the reply. I will process your comments deeply a bit later (especially wording and typos). 

Now some info and questions on the fly:
As mentioned Testing above, I've tested the instructions using [this](https://github.com/juri-sinitson/jurisin/tree/cypress-code-coverage-doc) branch of my testing repository. Pleas check the [package.json](https://github.com/juri-sinitson/jurisin/blob/cypress-code-coverage-doc/package.json) to see the exact versions. I'm using both Angular 13.x and Nx 13.x there. It worked well, thus I opened this PR. 

Are the versions of dependencies in the testing repository I used high enough for you?

What are exact criteria on `@jsdevtools/coverage-istanbul-loader` to be recommended for Angular developers?

It works well in the repository I use for testing with Angular 13.x and Nx 13.x. But it seems to disturb you and maybe other contributor(s)/repository owner(s) that its last commit is in 2020. 
I suppose you expected the dependencies of `@jsdevtools/coverage-istanbul-loader` be bumped up.

Is the something else you expect on this tool (e.g. certain percentage of code coverage)?

Reply:
I am not familiar with @jsdevtools/coverage-istanbul-loader and the package looks unmaintained as it's last commit was in 2020. We don't want to recommend a package that is unmaintained.

Reply:
> the package looks unmaintained as it's last commit was in 2020. We don't want to recommend a package that is unmaintained

Sure. 

I'm thiking about starting maintaining it again. Thus I ask you, what has to be done on that package, that you wold want to recommend it. That's why I'm talking about bumping up of the dependencies and the percentage of the code coverage. 
Is there something else you would expect on that package to recommend it?

Reply:
Make sure it works with webpack v5 since the deps of coverage-istanbul-loader say webpack 4. but other than that it's fine. I looked around and other loaders are even older.